### ISSUE TITLE
xarray concat got unexpected keyword argument 'attrs'

### ISSUE THREAD

**Issue by Lukas113** [i4Ds/Karabo-Pipeline#146]
When I use `imaging_rascil` in `how_to_use_karabo_example.ipynb` (localhost Ubuntu 20.04 & CSCS.daint), I got:

![image](https://user-images.githubusercontent.com/32157249/189372633-22c47796-fa84-4a85-9c19-703fe8bf1b85.png)

This is caused in xarray.core.concat.py 593

I am not sure why this occurs and how to approach it since in the `Imager` just contains the .ms observation and the error is thrown through RASCIL code. Maybe this is something RSCIL fixed in a newer release? I've just found an issue related to on  https://github.com/pydata/xarray/issues/6827 , but I'm not sure if this is what we are looking for.

**Comment by deiruch** [comment]
Can you provide a full stacktrace from imager.py to xarray?

It seems that whatever object we're deserializing here doesn't have the correct constructor.

**Comment by kenfus** [comment]
FYI: I get the same error

**Comment by kenfus** [comment]
Reinstalling another version fixes this:

`pip install xarray==2022.3.0`

A solution is just to fix this version in the meta.yaml.

Now the notebook-cell with 
```
imager_askap.ingest_chan_per_blockvis = 1
imager_askap.ingest_vis_nchan = 16

deconvolved, restored, residual = imager_askap.imaging_rascil(
..
)
```

works.

**Comment by kenfus** [comment]
Still fails with rascil==0.7.0. Will create a PR with an updated xarray version. However, Rascil is pinning the newest version (2022.6.0) in their requirements but i don't see an issue for this error in their repository. Do we call rascil somewhow different?

**Comment by kenfus** [comment]
@deiruch We saw that rascil released a hotfix version `0.7.1` which apparently deals with this issue. Would be better than fixing the verison.

**Comment by deiruch** [comment]
Great. Can't we both upgrade AND fix it?

**Comment by kenfus** [comment]
Sadly no, because creating a python package from rascil 0.7.1 gives an error. The pipeline works perfectly for 0.5.0 and 0.7.0 (and I guess all the other versions but not for 0.7.1). Maybe rascil did not update some parameters yet by mistake? @Lukas113 

So now we just downgrade xarray to 2022.3.0.

**Comment by Lukas113** [comment]
To me it seems that RASCIL has launched their upgrade, but didn't update at least their version number. So for RASCIL 0.7.1 we have to fix the dependencies first. So it seems to be a good idea to downgrade xarray for 0.7.0 and fix the dependencies of 0.7.1 as soon as possible.
